sections/discussion/taxonomy-topic.tex

```diff
@@ -6,6 +6,17 @@ \subsection{Taxonomy-topic model}\label{sec:taxonomy_analysis}
 This is a good feature that allows the model to apply an extra layer of preprocessing, automatically filtering away irrelevant words into topics.
 This feature is also seen in some other topic models, such as in the hierarchical \gls{lda} (hLDA) by \citet{hLDA2004} and the embedded topic model (ETM) by \citet{dieng2020topic}.
 
+Since this models deals with more topic distributions than the other models, it is worth checking whether it also converges within the first 50 epochs, as with \gls{lda}.
+This does seem to be the case, as indicated by \autoref{fig:pachinko_train}.
+Here it can be seen that the topic coherence curve has flattened significantly, and thus additional epoch will have diminishing returns.
```

Comment:
```suggestion
Here it can be seen that the topic coherence curve has flattened significantly, and thus additional epochs will have diminishing returns.
```